draft-ietf-taps-interface.md

```diff
@@ -978,6 +978,21 @@ Send Action. It is used to mark data safe for certain 0-RTT establishment
 techniques, where retransmission of the 0-RTT data may cause the remote
 application to receive the Message multiple times.
 
+### Final {#send-final}
+
+Final is a boolean property. If true, this Message is the final message that
+the application will send on a Connection. This allows underlying protocols
+to indicate to the Remote Endpoint that the Connection has been effectively
+closed in the sending direction. For example, TCP-based Connections can
+send a FIN once a Message marked as Final has been completely sent. 
+Protocols that do not support signalling the end of a Connection in a given
+direction will ignore this property.
+
+Note that a final Message must always be sorted to the end of a list of Messages.
+The final property overrides Niceness and any other property that would re-order
+Messages. If another Message is sent after a Message marked as final has already
+been sent on a Connection, the new Message will report an error.
+
```

Comment:
Capitalization of "Final" vs. "final" is inconsistent above - I suggest to write "The Final property overrides..." and "...marked as Final ..."  in the last two sentences above.